carting/management/commands/ingest_ouvrage_xml.py

```diff
@@ -1,27 +1,27 @@
 from xml.etree import ElementTree
+from pathlib import Path
 
 import requests
 from django.conf import settings
 from django.core.management.base import BaseCommand
 
 from carting.models import OuvrageSection
-from spo import generator
 
 
 class Command(BaseCommand):
     def add_arguments(self, parser):
         parser.add_argument(
             "ouvrage",
-            help="Ouvrage name, ex = 12, c22, l1",
+            help="Ouvrage xml file, ex = ./12.xml, ./c22.xml, ./l1.xml",
```

Comment:
On peut demander à `argparse` de créer directement un objet `Path` pour nous je crois. Et donc enlever des lignes au début de `handle`

```suggestion
            help="Ouvrage xml file, ex = ./12.xml, ./c22.xml, ./l1.xml",
            type=Path,
```